packages/core/src/App/Components/Elements/NotificationMessage/notification.jsx

```diff
@@ -177,6 +179,10 @@ Notification.propTypes = {
             'close_mx_mlt',
         ]).isRequired,
     }),
+    config: PropTypes.shape({
+        // Configurations controlling display of Notification banner
+        account_open_since_days: PropTypes.number,
```

Comment:
`notification.jsx` should be independent as possible. `account_open_since_days` is injecting a prop which is no way related to notification.